.github/pull_request_template.md

```diff
@@ -16,13 +16,16 @@
     of them would change the meaning incorrectly. The only changes to be made are to add a description text under the
     description heading and to add a 'x' to the correct box.
 -->
-    [ ] Fix
-    [ ] Refactor
-    [ ] Target update
-    [ ] Functionality change
-    [ ] Docs update
-    [ ] Test update
-    [ ] Breaking change
+    [ ] Patch update (Bug fix / Target update / Docs update / Test update / Refactor)
```

Comment:
We should link to contribution guidelines with some further description of these.

Reply:
I believe refactor should go into feature one as stated also in our guidelines

Reply:
> I believe refactor should go into feature one as stated also in our guidelines

Bartek thought it should go to patch unless the actual functionality is changed

Reply:
> We should link to contribution guidelines with some further description of these.

Don't we already do that in the non visible notes? Or are you saying we should make that note visible?

Reply:
Could be on the top as "Read this to understand pull request template" or similar note

Reply:
> Bartek thought it should go to patch unless the actual functionality is changed

@kjbracey-arm is that OK? I've seen this previously considered as refactors are internal changes (the worst thing there is introducing new bugs, but all backward compatible).

Reply:
The reason to not include refactors (or any change not fixing a specific bug) would be to reduce the chance of unneccessarily introducing new bugs.

It increases confidence for users that the patch series is safe to accept general updates from - if they're on 5.14.x, they could feel confident that 5.14.y won't break anything.

In semantic versioning terms, certainly, a refactor would be a patch version increment, as it's /supposed/ to be backwards compatible. But a Linux kernel patch release wouldn't accept it a backported kernel "refactor" unless it was to fix a specific bug, on grounds of risk.

So Bartek or someone needs to make the call on desired stability of backport patch series.

Reply:
@bulislaw Please confirm this change is fine .